### ISSUE TITLE
Can't delete cluster created with 0.10.2 with 0.11.0

### ISSUE THREAD

**Issue by benbrandt** [weaveworks/eksctl#1636]
**What happened?**
I tried to delete a cluster I created with 0.10.2:
```
eksctl delete cluster --name=eksworkshop-eksctl
```
And I got the following error message:
```
[ℹ]  eksctl version 0.11.0
[ℹ]  using region us-west-2
[ℹ]  deleting EKS cluster "eksworkshop-eksctl"
[✖]  failed to get Fargate profile(s) for cluster "eksworkshop-eksctl": AccessDeniedException: Account XXXX is not authorized to use this service
	status code: 403, request id: XXXXX
```

**What you expected to happen?**
The cluster to delete.

**How to reproduce it?**
- Create a cluster with the previous version of eksctl
- Upgrade eksctl
- Try to delete the previous cluster

**Anything else we need to know?**
Using Mac and the eksctl binary from homebrew

**Versions**
Please paste in the output of these commands:
```
$ eksctl version
[ℹ]  version.Info{BuiltAt:"", GitCommit:"", GitTag:"0.11.0"}
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.3", GitCommit:"b3cbbae08ec52a7fc73d334838e18d17e8512749", GitTreeState:"clean", BuildDate:"2019-11-14T04:24:34Z", GoVersion:"go1.12.13", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.8-eks-b8860f", GitCommit:"b8860f6c40640897e52c143f1b9f011a503d6e46", GitTreeState:"clean", BuildDate:"2019-11-25T00:55:38Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}
```

**Logs**
Include the output of the command line when running eksctl. If possible, eksctl should be run with debug logs. For example:
`eksctl get clusters -v 4`
Make sure you redact any sensitive information before posting.
If the output is long, please consider a Gist.

**Comment by eightlimbed** [comment]
Noticed this today, too :(

**Comment by benbrandt** [comment]
Yep just created a new one and can't delete it, created from 0.11.0

**Comment by cPu1** [comment]
This has been fixed in [0.11.1](https://github.com/weaveworks/eksctl/releases/tag/0.11.1). Please try it out.

**Comment by benbrandt** [comment]
It works! thanks!
